package.json

```diff
@@ -36,11 +36,13 @@
   "dependencies": {
     "eva.js": "^1.0.0",
     "highlight.js": "^9.7.0",
-    "uid": "0.0.2"
+    "uid": "0.0.2",
+    "vue": "~2.0.3"
```

Comment:
This was added due to a strange version conflict I was seeing between `vue` and `vue-template-compiler`. I'm assuming this is a mismatch of versions between `vbuild` and `eva.js`. 

![image](https://cloud.githubusercontent.com/assets/3087225/19500496/d6c5c97e-956b-11e6-8dc4-7f125e6a8257.png)